autoload/codi.vim

```diff
@@ -513,8 +513,13 @@ function! s:codi_handle_data(data, msg)
       if a:data['received'] > a:data['expected']
         call s:log('All prompts received')
         call s:stop_job_for_buf(a:data['bufnr'])
-        silent call s:codi_handle_done(
-              \ a:data['bufnr'], join(a:data['lines'], "\n"))
+        if s:nvim && g:codi#virtual_text
```

Comment:
Similar comment as above - can we extract this check into a meaningfully-named function?